lib/src/main/java/com/jesusm/holocircleseekbar/lib/HoloCircleSeekBar.java

```diff
@@ -313,22 +328,30 @@ protected void onDraw(Canvas canvas) {
 
         canvas.translate(mTranslationOffset, mTranslationOffset);
 
+        // Draw the pointer's "halo"
+        canvas.drawCircle(pointerPosition[0], pointerPosition[1],
```

Comment:
This code should be placed near line 348 as it's drawing a similar part of the canvas the code in that line does.